src/views/communities/components/AddCommunityDialog.js

```diff
@@ -0,0 +1,343 @@
+                  required
+                  sx={{ height: 50 }}
+                >
+                  {users.map((user) => (
+                    <MenuItem key={user.id} value={user.username}>
+                      {user.username}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </FormControl>
+            </Grid>
+          </Grid>
+          <Grid container spacing={2}>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                fullWidth
+                margin="normal"
+                id="communityName"
+                name="communityName"
+                label="Community Name"
+                variant="outlined"
+                placeholder="Ismaili Mubarak Scout Group"
+                value={formData.communityName}
+                onChange={handleInputChange}
+                required
+                InputProps={{
+                  sx: { height: 50 }
+                }}
+              />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                fullWidth
+                margin="normal"
+                id="slug"
+                name="slug"
+                label="Slug (Unique Identity)"
+                variant="outlined"
+                placeholder="scouts-mubarak321"
+                value={formData.slug}
+                onChange={handleInputChange}
+                error={!!slugError}
+                helperText={slugError}
+                required
+                InputProps={{
+                  sx: { height: 50 }
+                }}
+              />
+            </Grid>
+          </Grid>
+          <TextField
+            fullWidth
+            margin="normal"
+            id="description"
+            name="description"
+            label="Description"
+            variant="outlined"
+            placeholder="Enter a description for the community"
+            value={formData.description}
+            onChange={handleInputChange}
+            required
+            multiline
+            rows={4}
+            sx={{ mt: 2 }}
+          />
+
+          <Box display="flex" justifyContent="flex-end" gap={2} mt={2}>
+            <Button
+              variant="outlined"
+              onClick={() => {
+                clearForm();
+                handleClose();
+              }}
+            >
+              Cancel
+            </Button>
+            <Button variant="contained" color="primary" type="submit" disabled={isSubmitting}>
+              {isSubmitting ? 'Adding...' : 'Add Community'}
+            </Button>
+          </Box>
+        </Box>
+      </Box>
+    </Modal>
+  );
+};
+
+const App = () => {
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <div>
+      <Button sx={{ color: 'white' }} onClick={handleOpen} fullWidth size="large" type="submit" variant="contained" color="secondary">
```

Comment:
Add one button on dashboard or somewhere else for now 
and open this dialog on click of that button

button should not be created inside Dialog component